.github/actions/install-chainweb-from-artifacts/action.yml

```diff
@@ -16,8 +16,11 @@ runs:
       shell: bash
       run: |
         case "${{ inputs.os }}" in
-        ubuntu*)
-          sudo apt-get install -y build-essential libgflags-dev libtbb-dev
+        ubuntu-20.04)
+          sudo apt-get install -u libtbb2 libgflags2.2
+          ;;
+        ubuntu-22.04)
```

Comment:
Not sure. The libtbb version is different: libtbb2 vs. libtbb12. We could also use the libtbb-dev package (as we did before) which has the same name on both versions. But I thought, it may be better to to precise about the devs.